src/functionalTests/resources/json/consented/approved-consent-order-with-applicant-solicitor.json

```diff
@@ -0,0 +1,282 @@
+{
```

Comment:
I wonder if we shouldn't simplify these files with only must-have fields to make it clear what we are testing.

Reply:
good point but we would need an archeologist for that :-D